diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/LineLengthWarnText.kt

```diff
@@ -0,0 +1,208 @@
+package org.cqfn.diktat.ruleset.chapter3
+
+import com.pinterest.ktlint.core.LintError
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.ruleset.constants.Warnings.LONG_LINE
+import org.cqfn.diktat.ruleset.rules.DIKTAT_RULE_SET_ID
+import org.cqfn.diktat.ruleset.rules.LineLength
+import org.cqfn.diktat.util.lintMethod
+import org.junit.Test
+
+
+class LineLengthWarnText {
```

Comment:
```suggestion
class LineLengthWarnTest {
```
Seems like due to the [wrong name](https://maven.apache.org/surefire/maven-surefire-plugin/examples/inclusion-exclusion.html) your test hasn't been picked up by surefire plugin, therefore code coverage is so low.